docs/en/observability/ci-cd-observability.asciidoc

```diff
@@ -427,3 +427,53 @@ The Concourse CI doesn’t report health metrics through OpenTelemetry. However,
 https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanmetricsprocessor#span-metrics-processor[OpenTelemetry Collector Span Metrics Processor]
 to derive pipeline execution traces into KPI metrics like throughput and the error rate
 of pipelines.
+
+
+[discrete]
+[[check-service-health-from-cd-pipelines]]
+== Check service health from deployment pipelines
+
+
+Integrating automated service health checks in deployment pipelines is critical for end-to-end deployment automation, which crucially enables deployment frequency to be increased.
+
+Elastic Observability exposes HTTP APIs to check the health of services. You can integrate these APIs in deployment pipelines to verify the behaviour of newly deployed instances, and either automatically continue the deployments or roll back according to the health status.
+
+The following example shows a canary deployment pipeline that leverages Elastic health check HTTP APIs to automate the quality check before rolling out the deployment from the canary to the entire set of instances:
+
+image::images/ci-cd-canary-deployment-pipeline.png[Canary Deployment Pipeline]
+
+The health check can be performed invoking the `KIBANA_URL/internal/apm/services` API to compare the transaction error rate of the service on the newly deployed instances with a threshold value passing the following parameters to the invocation:
```

Comment:
@dedemorton can you please review this new sentence?

Reply:
This is a bit of a mouthful. I would at least break it down into a couple of sentences and avoid passive voice. How about something like:

Perform the health check by invoking the `KIBANA_URL/internal/apm/services` API to compare the transaction error rate of the service on the newly deployed instances with a threshold value. Pass the following parameters to the invocation:

Reply:
proposal adopted, thanks.